### ISSUE TITLE
Readme contains markdown examples

### ISSUE THREAD

**Issue by jameshosken** [shiffman/Open-Source-Studio-Playground#3]
Should we move markdown examples to a separate file (or link to [GitHub Markdown](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf) or the [Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet
), and keep the Readme as a more general overview of the playground?

**Comment by WenheLI** [comment]
Yes I think we should, a separate file will make the repo well organized  and we do need a markdown cheat sheet!

**Comment by D31F43** [comment]
@jameshosken 

Well, it is a great idea that writes some brief manual about markdown here. But I do recommend to have a glance of a better introduction written by Github [in this guide](https://guides.github.com/features/mastering-markdown/).

btw, I like your work. 😁

**Comment by shiffman** [comment]
Maybe related to #9? Resources could include helpful tips on Markdown.

**Comment by shiffman** [comment]
Guide about Markdown could live in the new Resources repo?

https://github.com/Open-Source-Studio-at-ITP/Resources
